synapsis/src/main/resources/templates/pages/home.html

```diff
@@ -16,6 +16,9 @@
             <button class="text-2xl btn btn-primary mt-8 ml-6">
                 Register
             </button>
+            <a class="text-2xl underline mt-8 ml-6" th:href="@{/admincreationpage}">
+                Create Admin (Restricted)
```

Comment:
We'll need to modify this at some point in a future story (perhaps when we focus more on the styling of the site) to only appear if the user is signed in as an admin. For now, we can leave it like this as we iterate.

Reply:
Yes, we can do that when we figure out more of the admin functionality in the later sprints. We might also need a separate homepage for the admin.